datadog_checks_base/datadog_checks/base/checks/base.py

```diff
@@ -28,6 +28,7 @@
     using_stub_aggregator = True
 
 from ..config import is_affirmative
+from ..constants import ServiceCheck
```

Comment:
why move this to a new file?

Reply:
Well it's a constant so it makes sense. Also there are others below like `ONE_PER_CONTEXT_METRIC_TYPES` that we should eventually move there as well

Reply:
Being able to access them from base seems very handy.

Reply:
Yea, that makes sense. But since these all still need to be members of the class it would just cut down on duplication in the AgentCheck classes, I suppose